Uppsala/UppEt1.xml

```diff
@@ -0,0 +1,469 @@
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Loefgren1974Katalog"/>
+                                            <citedRange unit="page">31–35</citedRange>
+                                            <citedRange unit="number">4</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Zettersteen1899Upsala"/>
+                                            <citedRange unit="page">508–510</citedRange>
+                                            <citedRange unit="number">1</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:1720Sveciae"/>
+                                            <citedRange unit="page">242</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:BmWebsite"/>
+                                        </bibl>
+                                    </listBibl>
+                                    <listBibl type="secondary">
+                                        <bibl>
+                                        <ptr target="bm:Bausi2015Wansleben"/>
+                                        <citedRange unit="page">214</citedRange>
+                                        <citedRange unit="number">58</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    
+                </msDesc>
+
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage><language ident="en">English</language></langUsage>
+            <langUsage><language ident="gez">Gəʿəz</language></langUsage>
+            <langUsage><language ident="de">German</language></langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2025-01-14">Created XML record</change>
+        </revisionDesc>
+    </teiHeader>
+    <text xml:base="https://betamasaheft.eu/">
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:hasUsed" active="UppEt1" passive="PRS10036Wansleb"/>
```

Comment:
I believe `saws `relations all describe work-work relations (if a work uses a passage from another work)

Reply:
Actually our schema says work-work but saws ontology says persons are ok 
https://ancientwisdoms.ac.uk/media/ontology/sawsOntologyV2.0.owl.xml#hasUsed
but probably the syntax should be inverted, not the manuscript used Wansleben, Wansleben used the manuscript
(active = subject, relation = predicate, passive = object)